src/prowjobsscraper/event.py

```diff
@@ -149,9 +149,9 @@ def index_equinix_usages(self, usages: list[EquinixUsage]):
 
     def scan_build_ids_from_index(self, index_prefix_singular: str) -> set[str]:
```

Comment:
can we take the opportunity to replace `index_prefix_singular` by `index_prefix`?